modules/components/item/Group.jsx

```diff
@@ -217,6 +217,7 @@ export class BasicGroup extends PureComponent {
         {...this.extraPropsForItem(item)}
         key={item.get("id")}
         id={item.get("id")}
+        groupId={props.id}
         //path={props.path.push(item.get('id'))}
```

Comment:
groupId should be passed to children here

Reply:
confused a little. Think u right, thanks for fixing me